odoo/addons/base/models/ir_http.py

```diff
@@ -204,6 +199,8 @@ def _pre_dispatch(cls, rule, args):
             if not isinstance(val, models.BaseModel):
                 continue
 
+            # Replace uid and lang placeholder by the current request.env.uid and request.env.lang
+            args[key] = val.with_env(request.env)
```

Comment:
Nitpicking, add an empty line so we understand that the comment is only about L203 :pray: 
```suggestion
            # Replace uid and lang placeholder by the current request.env.uid and request.env.lang
            args[key] = val.with_env(request.env)

```